Neos.Neos/Documentation/ExtendingNeos/CustomEditors.rst

```diff
@@ -3,48 +3,136 @@
 Custom Editors
 ==============
 
-Like with validators, using custom editors is possible as well. Every dataType has its default editor set, which
-can have options applied like::
-
-	Neos:
-	  Neos:
-	    userInterface:
-	      inspector:
-	        dataTypes:
-	          'string':
-	            editor: 'Neos.Neos/Inspector/Editors/TextFieldEditor'
-	            editorOptions:
-	              placeholder: 'This is a placeholder'
+.. note:: For documentation on how to create inspector editors for the legacy Ember version of the user interface refer to the older versions of the documentation.
+
+
+Every dataType has its default editor set, which can have options applied like::
+
+.. code-block:: yaml
+
+  Neos:
+    Neos:
+      userInterface:
+        inspector:
+          dataTypes:
+            'string':
+              editor: 'Neos.Neos/Inspector/Editors/TextFieldEditor'
+              editorOptions:
+                placeholder: 'This is a placeholder'
 
 On a property level this can be overridden like::
 
-	Neos:
-	  Neos:
-	    userInterface:
-	      inspector:
-	        properties:
-	          'string':
-	            editor: 'My.Package/Inspector/Editors/TextFieldEditor'
-	            editorOptions:
-	              placeholder: 'This is my custom placeholder'
-
-Namespaces can be registered like this, as with validators::
-
-	Neos:
-	  Neos:
-	    userInterface:
-	      requireJsPathMapping:
-	        'My.Package/Editors': 'resource://My.Package/Public/Scripts/Inspector/Editors'
-
-Editors should be named `<SomeType>Editor` and can be referenced by `My.Package/Inspector/Editors/MyCustomEditor`
-for example.
-
-Registering specific editors is also possible like this::
-
-	Neos:
-	  Neos:
-	    userInterface:
-	      inspector:
-	        editors:
-	          'Neos.Neos/BooleanEditor':
-	            path: 'resource://Neos.Neos/Public/JavaScript/Content/Inspector/Editors/BooleanEditor'
+.. code-block:: yaml
+
+  Neos:
+    Neos:
+      userInterface:
+        inspector:
+          properties:
+            'string':
+              editor: 'My.Package/Inspector/Editors/TextFieldEditor'
+              editorOptions:
+                placeholder: 'This is my custom placeholder'
+
+In order to implement a custom inspector editor one has to use the UI extensibility layer exposed through `@neos-project/neos-ui-extensibility` package.
+See :ref:`ui-extensibility` for the detailed information on the topic.
+
+Let's create a simple colour picker editor.
```

Comment:
Not an issue, but maybe it would be good to decide whether the manuals use AE or BE

Reply:
I prefer BE (Neos Foundation is even a British entity!), but everyone around seems to favor AE...